packages/extension-points/README.md

```diff
@@ -0,0 +1,36 @@
+# @drupal/extension-points-bundler
+
+Tooling to bundle and transpile your JavaScript to be used with the AdminUI.
+
+## Installation
+
+Add this module to your project. All the bundling and transpile dependencies and settings are included.
+
+```
+yarn add @drupal/extension-points-bundler
+
+# Build
+./node_modules/.bin/js-drupal-build
```

Comment:
how come this is `js-drupal-build`? I would have expected to be able to do `yarn extension-points-bundler`

Reply:
Seems like  a good idea to name them the same!

Reply:
adressed